component-catalog-app/Examples/Select.elm

```diff
@@ -82,38 +83,58 @@ example =
                 }
             , Heading.h2 [ Heading.plaintext "Example" ]
             , Select.view label attributes
-                |> Html.Styled.map ConsoleLog
+                |> Html.Styled.map ChangedTheSelectorValue
+            , Text.mediumBody
+                [ Text.plaintext <|
+                    """
+                    Note that if the value is bound (and why would you ever make a Select where it isn't?)
+                    then changing the list of options will not change its value.
+                    Furthermore, Select will only fire an event when a new, non-default value is selected.
+                    The current value is
+                """
+                        ++ getABetterNameToString state.selectedValue
+                        ++ "."
+                        ++ """
+                   if you change the "choices" above to "Overflowing text choices" there is no way to change the current value
+                   to Zaphod (the value of the single option with a very long name).
```

Comment:
Maybe it's worth changing the value/overflowing-text pair to be more obviously related to each other? Maybe change the label to be the full verse of the sonnet, and then have the value be something corresponding? Like `Shakespeare` or something? or something else entirely that is related?

Reply:
Or maybe... I think the 🐛 here that prevents the overflowing text from being selectable is in the modeling: having 2 discrete lists of options that aren't displayed at the same time, but are actually all part of the same union type.

Perhaps instead it would make sense to remove the option to use "short choices" or not and instead always include a long long long piece of text in the singular option list. Does that make sense?